runtime-common/core/core-types/decl/optional.h

```diff
@@ -64,6 +65,10 @@ class OptionalBase {
     return value_state_;
   }
 
+  explicit operator std::optional<T>() const noexcept {
```

Comment:
Can we get rid of this change? I don't see any benefit of having this thing in runtime